includes/abilities-api/class-wp-ability.php

```diff
@@ -269,10 +269,10 @@ public function get_meta(): array {
 	 *
 	 * @since 0.1.0
 	 *
-	 * @param array<string,mixed> $input Optional. The input data to validate.
+	 * @param mixed $input The input data to validate.
 	 * @return true|\WP_Error Returns true if valid or the WP_Error object if validation fails.
 	 */
-	protected function validate_input( array $input = array() ) {
+	protected function validate_input( $input ) {
```

Comment:
Nit-pick: Not strictly necessary, but for clarity I think we should make `$input` optional even on the internal methods.
```suggestion
	protected function validate_input( $input = null ) {
```

Reply:
I wasn't sure about that, but I'm happy to propagate the default as `null` in all necessary places 👍🏻